src/screens/Onboarding/OnboardingItem.tsx

```diff
@@ -69,21 +35,37 @@ const OnboardingItem: React.FC<Props> = ({ id, title, description, image }) => {
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: "#fff",
-    alignItems: "center",
     justifyContent: "center",
   },
   image: {
-    flex: 0.7,
+    flex: 0.6,
     justifyContent: "center",
+    marginTop: 60,
+  },
+
+  // slide: {
+  //   flex: 1, // Take up all screen
+  //   justifyContent: "center", // Center vertically
+  //   alignItems: "center", // Center horizontally
+  // },
+
+  header: {
+    ...FONTS.h3,
+    color: COLORS.primary,
```

Comment:
@Alphonce-Mutebi @Chegemartinx this font weight (line 55) should be defined in a font style, right? not here..

Reply:
@Alphonce-Mutebi i misplaced this comment, pls check line 55. It still sets the weight to bold.